fetch.bs

```diff
@@ -4400,12 +4406,17 @@ steps:
 
       <p class=note>This covers the case of <var>response</var> being a <a>network error</a>.
 
+     <li><p>Let <var>responseStatus</var> be 0 if <var>fetchParams</var>'s
+     <a for="fetch params">request</a>'s <a for=request>mode</a> is "<code>navigate</code>" and
+     <var>response</var>'s <a for=response>has-cross-origin-redirects</a> is true; otherwise
+     <var>response</var>'s <a for=response>status</a>.
```

Comment:
I was about to merge this, but I don't understand how this protects the response status when mode isn't "navigate". Shouldn't we be using the timing allow passed flag in that case?

Reply:
As noted in https://github.com/whatwg/fetch/pull/1468#discussion_r931266882, wouldn't the response be a filtered response here thereby protecting the response status?